R/utils.R

```diff
@@ -190,3 +190,13 @@ readr_enquo <- function(x) {
   }
   x
 }
+
+check_string <- function(x, nm = deparse(substitute(x)), optional = FALSE) {
+  if (rlang::is_string(x)) {
```

Comment:
I appreciate that this also catches `NA_character_`, which the previous checks would have allowed